base/features/kotlin/skeleton/gradle-build/gradle.properties

```diff
@@ -1,2 +1,3 @@
 micronautVersion=@version@
+micronautDataVersion=1.0.0.M4
```

Comment:
latest version is 1.0.0.M5 and I don't think we need this here

Reply:
this was a leftover from something i tried earlier i missed, it has been removed